src/appleseed.python/bindscene.cpp

```diff
@@ -58,10 +58,9 @@ void bind_scene()
     bpy::class_<Scene, auto_release_ptr<Scene>, bpy::bases<Entity, BaseGroup>, boost::noncopyable>("Scene", bpy::no_init)
         .def("__init__", bpy::make_constructor(create_scene))
         .def("get_uid", &Identifiable::get_uid)
-        .def("set_camera", &Scene::set_camera)
-        .def("get_camera", &Scene::get_camera, bpy::return_value_policy<bpy::reference_existing_object>())
         .def("get_environment", &Scene::get_environment, bpy::return_value_policy<bpy::reference_existing_object>())
         .def("set_environment", &Scene::set_environment)
+        .def("cameras", &Scene::cameras, bpy::return_value_policy<bpy::reference_existing_object>())
```

Comment:
Maybe move this line above the binding of `get_environment` to keep environment-related stuff together?